ui/src/common/components/CitationSummaryTable/CitationSummaryTable.jsx

```diff
@@ -94,14 +103,20 @@ class CitationSummaryTable extends Component {
                 <tr>
                   <th>Citations/paper (avg)</th>
                   <td>
-                    {(
-                      citeableBucket.getIn(['average_citations', 'value']) || 0
-                    ).toFixed(1)}
+                    <FormattedNumber>
+                      {(
+                        citeableBucket.getIn(['average_citations', 'value']) ||
+                        0
+                      ).toFixed(1)}
+                    </FormattedNumber>
                   </td>
                   <td>
-                    {(
-                      publishedBucket.getIn(['average_citations', 'value']) || 0
-                    ).toFixed(1)}
+                    <FormattedNumber>
+                      {(
+                        publishedBucket.getIn(['average_citations', 'value']) ||
```

Comment:
it's not introduced in this PR, but just realized `||`, `getIn` already has `unSetValue`, these could be written as `.getIn([...], 0)`

incase you'd like to improve.